cmd/antrea-controller/controller.go

```diff
@@ -84,7 +84,9 @@ func run(o *Options) error {
 
 	informerFactory.Start(stopCh)
 
-	controllerMonitor := monitor.NewControllerMonitor(crdClient, nodeInformer)
+	controllerMonitor := monitor.NewControllerMonitor(crdClient, nodeInformer, addressGroupStore,
+		appliedToGroupStore,
+		networkPolicyStore)
```

Comment:
nit: the arguments should be either in one line or an argument per line.

Reply:
Have changed the input parameter to NetworkPolicyInfoQuerier. Please help take a look.